q2_sample_classifier/classify.py

```diff
@@ -214,7 +216,10 @@ def regress_samples(ctx,
 
     accuracy_results, = scatter(predictions, metadata, 'ignore')
 
-    return sample_estimator, importance, predictions, summary, accuracy_results
+    _heatmap, _ = heatmap(ctx, table, importance)
```

Comment:
let's drop heatmaps from the regress_samples pipeline (sorry for change of decision, your PR helped me user-test this), see comments for more details.